tests/dft-grad-lr2/input.dat

```diff
@@ -25,7 +25,7 @@ set {
     dft_radial_points     120
     dft_spherical_points  434
     e_convergence         12
-    d_convergence         12
+    d_convergence         10
```

Comment:
if the test is turned off for C1, what motivated the change?

Reply:
good point. I fixed it with the 10 (last single-point of last gradient was failing to converge w/i iterations), then suppressed it anyways. Unsuppressing soon.